### PR TITLE
add nginx.md and update laravel server requirements

### PR DESCRIPTION
**Pull request by mmhyamin**

### CODE REVIEW

File: start/sw/web-app/nginx.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+sudo systemctl enable nginx
+# Start NGINX
+sudo systemctl start nginx
+# Check NGINX Status
+sudo systemctl status nginx
+```
+
+Now visit [http://localhost/](http://localhost/). You should see NGINX Test Page:
+
+![NGINX Test Page on Fedora](https://i.imgur.com/CoQxbD7.png)
```
Comment (pvalena) [inline start/sw/web-app/nginx.md]: Please upload(and reference) the image into this repository instead.

File: start/sw/web-app/nginx.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+---
+
+# What is NGINX
+
+NGINX is a free, open-source, high-performance HTTP server and reverse proxy, as well as an IMAP/POP3 proxy server. NGINX is known for its high performance, stability, rich feature set, simple configuration, and low resource consumption. Website: [nginx.org](https://nginx.org/), [nginx.com](https://www.nginx.com/).
+
+## Install NGINX Web Server
+
+```bash
+sudo dnf install nginx
```
Comment (pvalena) [inline start/sw/web-app/nginx.md]: Please use the `$` prepending each command([reference](http://developer-portal.github.io/contributing/write-introduction.html)).

File: start/sw/web-app/nginx.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
+---
+title: NGINX HTTP Server
+subsection: web-app
+order: 7
+---
+
+# What is NGINX
+
+NGINX is a free, open-source, high-performance HTTP server and reverse proxy, as well as an IMAP/POP3 proxy server. NGINX is known for its high performance, stability, rich feature set, simple configuration, and low resource consumption. Website: [nginx.org](https://nginx.org/), [nginx.com](https://www.nginx.com/).
```
Comment (pvalena) [inline start/sw/web-app/nginx.md]: `NGINX is a free...` - Fedora is free. Is it relevant to stress this here?

Comment (pvalena) [inline start/sw/web-app/nginx.md]: Same case with `open-source`.

File: start/sw/web-app/laravel5.md [hunk 4]
```diff
@@ -9,7 +9,7 @@ order: 4
 
 Laravel is a high-level PHP Web framework. To install Laravel the following needs to be satisfied:
 
-1. PHP >= 5.5.9
```
Comment (pvalena) [inline start/sw/web-app/laravel5.md]: PHP from Fedora is installed. Is it relevant to mention the version? Why not rather remove it altogether?

**Comment by pvalena** [comment]
Otherwise it LGTM.

Thank you for your PR!

**Comment by pvalena** [comment]
Thanks; the changes look good, but please remove(by squashing etc.) `fix pull #202` commit and amend the previous two commits.

**Comment by pvalena** [comment]
The easiest way would me IMHO to `git reset HEAD^^` and then add 2 respective commits.

**Comment by mmhyamin** [comment]
Apache Test Page on Fedora screenshot link is broken. Should I include it here?

**Comment by pvalena** [comment]
No, thanks, I will fix it.

**Comment by pvalena** [comment]
LGTM, Thanks a lot!

**Pull request merged**